internal/controller/utils.go

```diff
@@ -74,3 +78,22 @@ func paramsToEnv(params map[string]intstr.IntOrString) []corev1.EnvVar {
 	}
 	return envs
 }
+
+func CreateKubernetesClient() (*kubernetes.Clientset, error) {
```

Comment:
Client setup should happen in `main()` and not be hidden inside of pkg helper functions. As a reader, I would like to be able to open `main.go` and see all configuration settings then trace those settings through to all dependency clients as they are constructed, then see those clients passed into Reconciler instances, etc. We have diverged from this with some of the AutoConfigure stuff we are doing in the `cloud` pkg, but we should not continue expanding in that direction where its not necessary. I will likely refactor some of the cloud stuff to get use closer to this described pattern later.

Example: https://github.com/ardanlabs/service/blob/master/app/services/sales-api/main.go

Reply:
this whole function was removed. So fixed.